src/pages/index.tsx

```diff
@@ -1,13 +1,21 @@
 import { useUser, SignUp, SignOutButton, SignInButton } from "@clerk/nextjs";
 import Head from "next/head";
+import { Fragment } from "react";
 
 import { api } from "~/utils/api";
 
 export default function Home() {
-  const hello = api.example.hello.useQuery({ text: "from tRPC" });
   const user = useUser();
+
+  //to fetch all data from DB
+  const {data, error, isLoading} = api.posts.getAll.useQuery();
+
+  // if (!data) { throw new Error ('Fetch failed, no data was found')};
+
+  const renderData = data?.map((post) => (<div key={post.id}>{post.content}</div>))
+  
   return (
-    <>
+    <Fragment>
```

Comment:
```suggestion
    <>
```

実際の開発では`<>`の書き方にて統一しているのでこちらでお願いいたします！
どちらでも機能、挙動は変わらないとは思いますが💦